src/main/java/EmptyTile.java

```diff
@@ -1,5 +1,6 @@
-package game2048;
+package src.main.java;
```

Comment:
Yeah this package I think can be removed completely because it's not in a source module. I think all of these classes are duplicated elsewhere right?